.github/workflows/android_build.yml

```diff
@@ -0,0 +1,48 @@
+name: Build on Android NDK
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build on Android NDK ${{ matrix.arch.name }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        arch: 
+          - { name: armeabi-v7a }
```

Comment:
As long as this test fails, I suggest to add smth like `allow-failure`. Maybe `continue-on-error` is the GH Action equivalent of the TravisCI `allow-failure` option.